tests/Http/Api/ReactionsApiTest.php

```diff
@@ -20,37 +19,39 @@ class ReactionsApiTest extends BrowserKitTestCase
     public function testPostingAndSoftDeleteForReaction()
     {
         // Create a post to react to.
-        $post = factory(Post::class)->create();
+        $signup = factory(Signup::class)->create();
+        $signup->posts()->save(factory(Post::class)->make());
+        $post = $signup->posts->first();
```

Comment:
Doesn't the post factory make a signup for you?

Reply:
Ah yes! I had thought I'd need to fix each of these manually at first, will revert. 🔙

Reply:
Done in 2f711e8.